package.json

```diff
@@ -30,10 +30,11 @@
   },
   "homepage": "https://github.com/kesne/babel-plugin-inline-react-svg#readme",
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
-    "babel-preset-airbnb": "^2.5.1",
-    "babel-preset-react": "^6.24.1",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/node": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-preset-airbnb": "vikr01/babel-preset-airbnb#feature/babel-7",
```

Comment:
this can't merge until we're using the actual package

Reply:
@stramel this one's merged! https://github.com/airbnb/babel-preset-airbnb/pull/37 thanks @ljharb